### ISSUE TITLE
Spectral range specification is not respected

### ISSUE THREAD

**Issue by CorentinGrimaldi** [radis/radis#81]
## Expected Behavior
When plotting a spectrum calculated over a given spectral range (for example 4160nm - 4190nm), the plot should show the spectrum between the specified wavelengths (i.e. 4160nm and 4190nm)

## Current Behavior
The plot shows the spectrum over a slightly shifted spectral range (4158.85nm - 4188.85nm).

## Steps to Reproduce (for bugs)

Minimal code:
```
s = calc_spectrum(wavelength_min=4160,   # nm
                             wavelength_max=4190,   # nm
                             Tgas=300,            # K
                             pressure=1.01325,       # bar
                             mole_fraction=1e-3,
                             isotope='1,2,3,4',
                             path_length=1,      # cm
                            databank='CDSD-HITEMP',
                            )
s.plot(wunit='nm')
```
## Context
Well if the spectrum is calculated over a spectral range shifted from the one given, it is not a huge problem, one can just ask the calcul over a larger spectral range. However if the spectrum is calculated over the given spectral range but then plotted over a shifted one it is more problematic...

**Comment by CorentinGrimaldi** [comment]
Actually works in the correct range if plotted in nmin vacuum `s.plot(wunit='nm_vac')`

Input specifies:

> wavelength_min: float(nm) or ~astropy.units.quantity.Quantity
> minimum wavelength. Wavelength in 'air' or 'vacuum' depending of the value of the parameter 'medium='

But I use the default `'medium=air' `

So there is an interface error here.

**Comment by CorentinGrimaldi** [comment]
I checked that the spectrum is correct. Lines are at the proper position. 
It's just an interface error when calculating the spectral range.

**Comment by erwanp** [comment]
Any news on this @CorentinGrimaldi  ?

**Comment by minouHub** [comment]
First of all there is no problem in cm-1. In nm, the waverange of the calculated spectrum is effectively shifted, see below.

- Input:

```
s = calc_spectrum(
                # 1900, 2300,         # cm-1
                wavelength_min=4160,   # nm
                wavelength_max=4190,   # nm
                  molecule='CO2',
                  isotope='1,2,3',
                  pressure=1.01325,   # bar
                  Tgas=700,           # K
                  mole_fraction=0.1,
                  path_length=1,      # cm
                  verbose=False,
                  wstep=0.01,
                  medium='vacuum'
                  )
w_range = s.get_wavelength()
w_range2 = s.get_wavenumber()
print('nm', w_range.min(),w_range.max())
print('cm-1', w_range2.min(),w_range2.max())
s.plot(wunit='nm')
```

- Output:
```
nm 4158.850805066924 4188.857667519326
cm-1 2386.6348448687354 2403.8548448691113
```

-----------------
I tought there was in problem with the warenum/waverange conversion but first of all, is it normal that the medium is not taken into account in this conversion?
```
if wavenum_min is None and wavenum_max is None:
        assert wavelength_max is not None
        assert wavelength_min is not None
        wavenum_min = nm2cm(wavelength_max)
        wavenum_max = nm2cm(wavelength_min)
```

**Comment by erwanp** [comment]
@gagan-aryan , once #214 is fixed can you check that this one is fixed too ? It seemed it was the same error. Two birds one stone :)

**Comment by erwanp** [comment]
Error is fixed, probably by #217
